File: 3110-score-of-a-string/3110-score-of-a-string.cpp
class Solution {
public:
    int scoreOfString(string s) {
        int sum = 0;
        for(int i = 0; i < s.size()-1; i++){
            int a = (int) s[i];
            int b = (int) s[i+1];
            sum += abs(a - b);
        }
        return sum;
    }
};

File: 3110-score-of-a-string/3110-score-of-a-string.java
class Solution {
    public int scoreOfString(String s) {
        int n = s.length();
        int score = 0;
        for(int i = 1; i < n; i++){
            score += Math.abs(s.charAt(i-1) - s.charAt(i));
        }
        return score;
    }
}